iambic/config/wizard.py

```diff
@@ -510,15 +512,21 @@ def secret_config_dir(self, secrets_file: str = "secrets.yaml") -> str:
         return f"{Path(self.config_path).parent.joinpath(secrets_file)}"
 
     async def set_config_details(self):
-        try:
-            self.config_path = str((await resolve_config_template_path(self.repo_dir)))
-        except RuntimeError:
-            self.config_path = f"{self.repo_dir}/iambic_config.yaml"
+        self.config_path = await self._get_config_path()
 
         if os.path.exists(self.config_path) and os.path.getsize(self.config_path) != 0:
             log.info("Found existing configuration file", config_path=self.config_path)
             try:
                 self.config = await load_config(self.config_path)
+            except NoCredentialsError:
+                question_text = (
+                    "We couldn't find your AWS credentials. "
+                    "Please estart the wizard with valid AWS credentials "
```

Comment:
Could you use the same message and log status (error vs info) as we use here? https://github.com/noqdev/iambic/blob/main/iambic/plugins/v0_1_0/aws/utils.py#L219